packages/theme-docs/src/index.js

```diff
@@ -47,13 +47,6 @@ const defaultConfig = {
       nuxt.options.content.dir = path.resolve(nuxt.options.rootDir, nuxt.options.content.dir || 'content')
       // Configure `static/ dir
       nuxt.options.dir.static = path.resolve(nuxt.options.rootDir, nuxt.options.dir.static || 'static')
-      // Configure `components/` dir
```

Comment:
@pi0 Why removing this?

The theme needs to load global components inside the theme itself and the ones from the `rootDir`.

Reply:
You are right. Using `rootDir` was little bit misleading. I've updated it to `__dirname`

Reply:
@pi0 This has to be `nuxt.options.rootDir`, @nuxt/content is already loading the theme components, here `__dirname` also resolves to the theme so user components are not loaded anymore.

Reply:
Hmm i not sure to unrestand. `rootDir` cannot point to two directories. `rootDir` should be same as `__dirname` (path to `node_modules/@nuxt/theme/docs/src`. User components are loaded by the other hook in main content module

Reply:
Since the `srcDir` is the theme, the main content module is loading the theme components, `rootDir` however loads the user components.

Reply:
You can try to create a component in the `docs/` for example, they won't be loaded with the `__dirname` change.

Reply:
ok you are right reverted :)